README.md

```diff
@@ -2,59 +2,52 @@
-To stop the app run `docker-compose down`.
-
-Settings for the app can be changed in `.env`.
-Options for `.env`:
-- BACKEND_WORKERS - number of [workers used by gunicorn](https://docs.gunicorn.org/en/latest/run.html#commonly-used-arguments)
-- ANDROMEDA_RGB_SATELLITE_URL - URL pointing to a CSV file for joining RGB data
-
-### Certbot Setup on AWS
-Certificates can be installed to the EC2 VM using Certbot with Nginx on pip following [EFF instructions](https://certbot.eff.org/instructions?ws=nginx&os=pip).
+(optional) If you want to use or test a different branch of the Andromeda repository, checkout the desired branch.
+```bash
+cd Andromeda/Andromeda
+git checkout <branch>
+cd ..
+```
+2. Build the Docker image.
 ```bash
-sudo python3 -m venv /opt/certbot/
-sudo /opt/certbot/bin/pip install --upgrade pip
-sudo /opt/certbot/bin/pip install certbot certbot-nginx
-sudo ln -s /opt/certbot/bin/certbot /usr/bin/certbot
+docker image build -t andromeda .
 ```
-And to install Nginx on the Amazon Linux 2023 (following [instructions](https://awswithatiq.com/how-to-install-nginx-in-amazon-linux-2023/)) and run Certbot:
+3. Run the Docker image.
 ```bash
-docker-compose down
-sudo dnf update -y
-sudo dnf install nginx -y
-sudo systemctl start nginx
-sudo certbot certonly --nginx
+docker container run -p 7860:7860 andromeda
 ```
-Following prompted agreements, registration, and specifying andromeda.imageomics.org, the certificates are installed to:
 
-Certificate: `/etc/letsencrypt/live/andromeda.imageomics.org/fullchain.pem`
+While Hugging Face handles SSL encryption automatically, other deployment environments require SSL to be set up manually. One common approach is using a reverse proxy like Nginx. Below are some resources and tutorials that provide guidance on setting up SSL with Nginx in a Docker environment:
 
-Key: `/etc/letsencrypt/live/andromeda.imageomics.org/privkey.pem`
+- [Configure HTTPS for an Nginx Docker Container (Stackify)](https://stackify.com/how-to-configure-https-for-an-nginx-docker-container/)​
+- [Setup SSL with Docker, NGINX and Lets Encrypt (Programonaut)​](https://www.programonaut.com/setup-ssl-with-docker-nginx-and-lets-encrypt/)
+- [Set up SSL on Docker, Nginx Container and Let's Encrypt (SSLWiki)](https://sslwiki.org/set-up-ssl-on-docker-nginx-and-lets-encrypt/)
+- [Install TLS/SSL on Docker Nginx Container With Let’s Encrypt (Dev.to)](https://dev.to/macelux/how-to-install-tlsssl-on-docker-nginx-container-with-lets-encrypt-34c5)
+- [NGINX Docker with SSL Encryption (Self-signed) by Mike Polinowski​](https://mpolinowski.github.io/docs/DevOps/NGINX/2020-08-27--nginx-docker-ssl-certs-self-signed/2020-08-27/)
 
-Finally, shut down Nginx on the system to avoid interference with the container:
-```bash
-sudo systemctl stop nginx
-docker-compose up -d
-```
+These tutorials cover a range of setups including self-signed certificates, Let's Encrypt, and Certbot. Choose the one (or more) that aligns well with your deployment scenario.
```

Comment:
Do we really need this section? To me it just adds more questions. Like why would I choose one of these tutorials over the other. How is it possible to combine them as the documentation describes? K8s clusters often automatically create certs and include a load balancer and users wouldn't need to do this. Who is this tutorial list for? For me choosing a reverse proxy, setting up certs, and deploying to the many possible environments is a huge topic that could be an entire book.

Reply:
That's what I'm trying to hone ... the original thought was wanting to provide _something_ to help the person who wants to put it somewhere other than Hugging Face, but it's too big a topic to adequately address. 

I think we can just say something like "Hugging Face handles SSL encryption automatically. Deploying in another environment may require additional configuration." Does that sound good, or just eliminate it altogether?

Reply:
The HF handles SSL for us part sounds good.

Reply:
I think it's fine to simply document how _we_ ended up doing this part when we needed it. That also addresses the  question of audience (which is a very good one), namely for that it's ourselves if we needed to do this again.

Reply:
Perhaps that level of detail gets moved to the User Guide though? Keeps the README a bit cleaner to have 
> Hugging Face handles SSL encryption automatically. Deploying in another environment may require additional configuration; previous configurations are discussed in [user-guide-section]().

Reply:
But that's not for users of the application; it would be for a sysadmin. There could of course be a guide for sysadmins, but if there are installation instructions here in the README already (which would also be more for admins than typical users), this doesn't strike me as an inappropriate place. Though there could just also be a page for admins in the wiki.

Reply:
So should we keep the docker components removed in [b173f27](https://github.com/Imageomics/Andromeda/pull/74/commits/b173f2733aa392ce3542617a06825a2731fe0e94) as well as the associated discussion in the README which was removed as part of [d39a868](https://github.com/Imageomics/Andromeda/pull/74/commits/d39a8684f0d2fefed6fcf9e76c449aaab38f1469)?

I may have acted too rashly based on John's comment on Slack:
>Since we aren't deploying to AWS anymore we could  consider if we should even keep the unused docker-compose config.

Reply:
To summarize, I think we should have 
> Hugging Face handles SSL encryption automatically. Deploying in another environment may require additional configuration; previous configurations are discussed in [Prior Configurations](https://github.com/Imageomics/Andromeda/wiki/Prior-Configurations).

Then on that page we can note something to the effect of
> The [Dockerfile](link to it) and [docker-compose.yml](link) used previously can be adapted to your use-case...

Or just point directly to the Release/commit where they both exist and say they can be found [here]().

Reply:
Yeah we can both delete stuff we no longer use _and_ continue to provide links to them. The joys of version control 😄